package.json

```diff
@@ -3,6 +3,12 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "android-mainnet-debug": "yarn run pre-packager && react-native run-android --variant=mainDebug",
```

Comment:
I would delete this. This is too hacky, because it doesn’t work for releases.

Reply:
So basically, I would focus in making it work for the `fastlane` deployment scripts